ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```
@@ -143,10 +143,28 @@ tests:
   commands: |
     export ROOT_DIR=/tmp
     export COVER_PROFILE=/tmp/reports/unit_coverage.out
+    export CODECOV_TOKEN=$(cat /tmp/secret/CODECOV_TOKEN)
```

I think the convention we took is having file names with lowercase, like this:
```suggestion
    export CODECOV_TOKEN=$(cat /tmp/secret/codecov_token)
```

---

okay, I'll change it here and in vault